### ISSUE TITLE
Managing deploy branches

### ISSUE THREAD

**Issue by kalilsn** [jeancochrane/bunny-hook#12]
Right now, if I understand correctly, the worker checks whether the last part of the url matches the branch name in the payload, as well as the branch name from the secrets file. With multiple repositories, this isn't going to work, since the secrets file is keyed by branch name. Additionally, with this strategy, if you want to deploy the same repository from two different branches, such as `staging` and `master`, you have to create a separate webhook on github, in addition to configuring the deploys for each.

I think we can solve both of these problems by keying the secrets file with the _repository name_ instead of the branch name, using the _repository name_ in the url (`/hooks/github/<repository_name>`), and configuring the different branches in the yml file. Additionally, if we make this change, we can add support for events other than push, such as releases.

**Comment by jeancochrane** [comment]
Agreed! I like this change. Not supporting multiple repositories sounds like a bug to me, so it'd be great to get this in as part of https://github.com/jeancochrane/bunny-hook/pull/8 or pull out the portion of that PR that changes the secrets file format until this change can come in.
